cmd/run/run.go

```diff
@@ -267,7 +269,7 @@ func run(_ *cobra.Command, _ []string) {
 		panic(err)
 	}
 
-	logger := logger.MustNewLogger(config.Log.Format, config.Log.Level)
+	logger := logger.MustNewLogger(config.Log)
```

Comment:
It didn't feel right to add another method parameter when we have a dedicated struct already